app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,276 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS (and SCSS, if configured) file within this directory, lib/assets/stylesheets, or any plugin's
+ * vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Proxima Nova', sans-serif;
+}
+
+body {
+  background-color: #f5f6f8;
+}
+
+h2,
+h3,
+h5 {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+h1 {
+  text-align: center;
+}
+
+.text-align-center {
+  text-align: center;
+  align-self: center;
+}
+
+.flex-column {
+  display: flex;
+  flex-direction: column;
+}
+
+.first-field {
+  margin-top: 5px;
+}
+
+.new_user input {
+  border: none;
+  border-radius: 0;
+}
+
+.new-category {
+  display: flex;
+  flex-direction: column;
+  align-self: center;
+  width: 100%;
+  justify-content: space-between;
+  flex: 1;
+}
+
+.new-category input {
+  border: none;
+  border-radius: 0;
+}
+
+.save-catagory input {
+  width: auto !important;
+  height: auto !important;
```

Comment:
- [ ] Kindly don't use `!important`. If you use important too liberally, it can make it harder to override styles later on. This can create a situation where you need to use important even more, leading to a vicious cycle that makes your CSS harder to manage.